src/unit_tests/wazuh_db/test_wdb_global.c

```diff
@@ -4623,9 +4669,29 @@ void test_wdb_global_select_group_belong_exec_fail(void **state)
     expect_any_always(__wrap_sqlite3_bind_int, index);
     expect_any_always(__wrap_sqlite3_bind_int, value);
     will_return_always(__wrap_sqlite3_bind_int, SQLITE_OK);
-    will_return(__wrap_sqlite3_errmsg, "ERROR MESSAGE");
-    will_return(__wrap_wdb_exec_stmt_single_column, NULL);
-    expect_string(__wrap__mdebug1, formatted_msg, "wdb_exec_stmt(): ERROR MESSAGE");
+    /* wdb_exec_stmt_sized */
+    wrap_wdb_exec_stmt_sized_failed_call(STMT_SINGLE_COLUMN);
+
+    expect_string(__wrap__mdebug1, formatted_msg, "Failed in getting the groups of the agent.");
```

Comment:
Suggestion: "Failed to get the agent groups"

Reply:
Fixed !